src/get-logical-id.js

```diff
@@ -8,6 +8,10 @@ module.exports = function getLogicalID (pathToCode) {
     // Websocket logical ids are prefixed with "Websocket"
     logicalID = 'Websocket' + logicalID
   }
+  else if (pathToCode.includes('/tables/')) {
+    // DynamoDB Stream logical ids are suffixed with "Stream"
+    logicalID = logicalID + 'Stream'
```

Comment:
Just a heads up, as part of the Inventory refactors I made the Lambda resource names more explicit, deterministic, and impossible to conflict across pragmas: https://join.slack.com/t/architecture-as-text/shared_invite/MjE2MzU4Nzg0NTY1LTE1MDA2NzgyMzYtODE2NzRkOGRmYw